index.js

```diff
@@ -134,6 +134,8 @@ module.exports = {
     "spaced-comment": ["warn", "always", { "exceptions": ["-", "+"] }],
     'no-console': 'warn',
     'default-param-last': 'warn',
+    "curly": ['warn', "all"],
+    'prefer-arrow-callback': 'warn',
```

Comment:
This is prefer like this. 
```ts
foo(function(a) { return a; }); // ERROR
// prefer: foo(a => a)

foo(function() { return this.a; }.bind(this)); // ERROR
// prefer: foo(() => this.a)
```

ref: https://eslint.org/docs/latest/rules/prefer-arrow-callback